exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/metric_exporter/__init__.py

```diff
@@ -137,7 +139,27 @@ def __init__(
                 ObservableUpDownCounter: AggregationTemporality.CUMULATIVE,
                 ObservableGauge: AggregationTemporality.CUMULATIVE,
             }
+
+        elif otel_exporter_otlp_metrics_temporality_preference == "LOWMEMORY":
```

Comment:
Should we use an enum to represent these now that the list is growing?

Reply:
We have logic in our distros that check of type of temporality. It would be better to refer to an enum rather than a hardcoded string.

Reply:
Sorry, where in our distros we check for temporality type? Also, these strings here (`"LOWMEMORY"`, `"DELTA"` and `"CUMULATIVE"`) are not temporality types but values for an environment variable.

Reply:
Sorry I meant to say our Azure distros. Regardless of if we are doing it in our current code our not, we probably should reduce the amount of hardcoded strings in our code right? (it doesn't even have to be an enum, could be just a class that defines known temporality preferences)

Reply:
I'd say we don't create anything else for the time being. We are in the same situation for any other environment variable that uses string values so, this is consistent at least. I do agree that there must be a better way of doing this than just having strings in the modules where the environment variables are used but a more comprehensive solution is needed that also encompasses boolean, integers, floats, etc. since those are also values environment variables may have.